zhaquirks/quirk_ids.py

```diff
@@ -3,3 +3,8 @@
 # Tuya
 TUYA_PLUG_ONOFF = "tuya.plug_on_off_attributes"  # plugs with configurable attributes on the OnOff cluster
 TUYA_PLUG_MANUFACTURER = "tuya.plug_manufacturer_attributes"  # plugs with configurable attributes on a custom cluster
+
+# Xiaomi
+XIAOMI_AQARA_VIBRATION_AQ1 = (
+    "xiaomi.aqara_vibration_aq1"  # vibration sensor with custom cluster handler
+)
```

Comment:
Okay, the line wrapping is unfortunate, but can't really be avoided unless the name is shortened and/or the comment is shortened.

I'd leave it like this for now and see if this file should be "restructured" (move comments, change to docstrings or whatever) in the future.